From 512f1b3a4727a3a18f8fa55f196eedcacdd4a872 Mon Sep 17 00:00:00 2001
From: Changqing Li <changqing.li@windriver.com>
Date: Wed, 18 Jul 2018 13:01:03 +0800
Subject: [PATCH] fix musl libc compile error

use op_basename instead of basename to fix below error:
../../oprofile-1.2.0/pp/oparchive.cpp:264:20: error: 'basename'
was not declared in this scope

Signed-off-by: Changqing Li <changqing.li@windriver.com>
---
 pp/oparchive.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/pp/oparchive.cpp b/pp/oparchive.cpp
index 6221e14..274152b 100644
--- a/pp/oparchive.cpp
+++ b/pp/oparchive.cpp
@@ -261,7 +261,7 @@ int oparchive(options::spec const & spec)
 		/* determine the session name of sample file */
 		int offset = sample_name.find('{');
 		string base_samples_dir = sample_name.substr(0, offset-1);
-		string session = basename(base_samples_dir.c_str());
+		string session = op_basename(base_samples_dir.c_str());
 		/* Get rid of the the archive_path from the name */
 		string sample_base = sample_name.substr(offset);
 		string sample_archive_file = dest_samples_dir + "/" + session + "/" + sample_base;
-- 
2.7.4

